content/artifacts/artifactory/plugin.js

```diff
@@ -7,537 +7,608 @@
 // * Deposit artifacts on your planets
 // * Find untaken planets with artifacts and jump to them
 
-import {
-  html,
-  render,
-  useState,
-  useLayoutEffect,
-} from 'https://unpkg.com/htm/preact/standalone.module.js';
-
-import {
-  BiomeNames,
-  energy,
-  coords,
-  isMine,
-  isUnowned,
-  unlockTime,
-  canWithdraw,
-  hasArtifact,
-  canHaveArtifact,
-  canFindArtifact,
-} from 'https://plugins.zkga.me/utils/utils.js';
-
-import {
-  Energy,
-  EnergyGrowth,
-  Defense,
-  Range,
-  Speed,
-} from 'https://plugins.zkga.me/game/Icons.js';
+const {
+    html,
+    render,
+    useState,
+    useLayoutEffect,
+} = await import('https://unpkg.com/htm/preact/standalone.module.js');
+
+const {
+    BiomeNames,
+    energy,
+    coords,
+    isMine,
+    isUnowned,
+    unlockTime,
+    canWithdraw,
+    hasArtifact,
+    canHaveArtifact,
+} = await import('https://plugins.zkga.me/utils/utils.js');
+
+const {
+    Energy,
+    EnergyGrowth,
+    Defense,
+    Range,
+    Speed,
+} = await import('https://plugins.zkga.me/game/Icons.js');
```

Comment:
This is the wrong import style. I just went through and updated all of these. Can't use this and expect to work on all browsers.

Reply:
Understood now, thank you.